.editorconfig

```diff
@@ -3,21 +3,16 @@ root = true
 [*]
```

Comment:
As spec says, `[*]` won't match every file in all subdirectories, but `[**]` will.

![image](https://user-images.githubusercontent.com/17182159/128342344-10a64ae8-bf59-49e0-bc7a-2b54319bac9f.png)